example/app/package.json

```diff
@@ -8,7 +8,7 @@
     "visualize": "meteor --production --extra-packages bundle-visualizer"
   },
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
+    "@babel/runtime": "^7.11.2",
```

Comment:
Should this have been downgraded?

Reply:
@GeoffreyBooth Hello, this was the result of a `meteor create --release 2.2.1 --blaze app`

`7.12.5` comes from this commit https://github.com/disney/meteor-base/commit/4d4456654eae85b250fec64e217b92e16276336b which is for Meteor 1.12.

Running the command `meteor create --release 1.12 app` gives me a `"@babel/runtime": "^7.11.2",` 

Not sure what is the best move to do here. I can revert this change if you want

Reply:
It had a newer version because I ran `npm update` in order to have the latest versions in the **package.json**. IMHO I always use the latest versions to ensure a project updated and find possible future bugs.

Reply:
Should I set it to 7.14.6 ? https://www.npmjs.com/package/@babel/runtime

Reply:
For me it would be ok, but the final decision rely on @GeoffreyBooth .

Reply:
I think what we want is for the _tests_ run on the versions of dependencies that are pinned for the particular version of Meteor being tested. But this example app isn’t part of the tests, if I remember correctly. So if that’s the case, then sure, we might as well always have the example use the latest versions as of whenever we update it.

Reply:
@GeoffreyBooth now it is updated and ready to merge :)